main/adapters/repository/init_sqlite.go

```diff
@@ -0,0 +1,109 @@
+package repository
+
+import (
+	"context"
+	"database/sql"
+	log "github.com/sirupsen/logrus"
+	"github.com/ubirch/ubirch-client-go/main/config"
+	"github.com/ubirch/ubirch-client-go/main/ent"
+)
+
+const MigrationVersionSqlite = "1.0.1"
+
+func MigrateToSqlite(c config.Config) error {
+	txCtx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	dbManager, err := NewSqliteDatabaseInfo(c)
+	if err != nil {
+		return err
+	}
+
+	tx, shouldMigrate, err := checkVersionSqlite(txCtx, dbManager)
+	if err != nil {
+		return err
+	}
+	if !shouldMigrate {
+		log.Infof("database migration version already up to date")
+		return nil
+	}
+
+	log.Println("database migration version updated, ready to upgrade")
+	identitiesToPort, err := getAllIdentitiesFromLegacyCtx(c)
+	if err != nil {
+		return err
+	}
+
+	if _, err := dbManager.db.Exec(CREATE[SQLiteIdentity]); err != nil {
```

Comment:
this has to be removed now since it is already done in NewSqliteDatabaseInfo(c)

Reply:
yes, done